src/math-functions.md

```diff
@@ -108,3 +108,10 @@
    - Second parameter denotes number of digits after floating point
      
      <img src="images/round1.png" width="200" />
+
+16. **Random**
+   - Declare integers x, y
+   - Take input for x, y  
+   - Choose assignment block and write the code `x = random(y)`
+     
+     <img src="images/random.png" width="200" />
```

Comment:
Did you intend to attach this file?

Reply:
![random](https://github.com/flowgrammers-org/flowgramming-manual/assets/86895240/e0569aa1-56ed-470b-9e9c-db655a4e372b)

Reply:
No, the file is not uploaded in that path in this PR.

Reply:
i created a pull request, thanks ;)